here-naksha-lib-psql/src/commonTest/kotlin/naksha/psql/DbReadWriteTest.kt

```diff
@@ -25,11 +25,11 @@ package naksha.psql
 //import naksha.model.request.condition.PRef.NON_INDEXED_PREF
 //import naksha.model.request.condition.SOp
 //import naksha.model.request.condition.SOp.Companion.intersectsWithTransformation
-//import naksha.model.request.condition.geometry.BufferTransformation.Companion.bufferInMeters
-//import naksha.model.request.condition.geometry.BufferTransformation.Companion.bufferInRadius
-//import naksha.model.response.ErrorResponse
-//import naksha.model.response.Response
-//import naksha.model.response.SuccessResponse
+//import naksha.model.request.condition.BufferTransformation.Companion.bufferInMeters
```

Comment:
please add TODO comment to address this file (delete test / repair it)

Reply:
actually I think you can mention *all* files in this package

Reply:
lib-psql is under Alex's and Pawel's decision, but I think they already marked these with TODOs, this file is marked line 615